src/api/lib/metrics/types.ts

```diff
@@ -0,0 +1,104 @@
+export interface IADSApiMetricsParams {
+  bibcode: string;
+}
+
+export interface ICitationsHistogram {
```

Comment:
I think you could simplify this a bit by switching the type to be a union on these strings

```
type CitationsHistogramKey = 'nonrefereed to nonrefereed' | 'nonrefereed to nonrefereed normalized' | ...;

export type CitationsHistogram = Record<CitationsHistogramKey, { year: number }>
```

Something like that anyway

Reply:
Updated